src/tests/Navigation.test.js

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import Navigation from '../components/Navigation';
+
+// mock Redux store
+const store = {
+  getState: () => ({}),
+  dispatch: () => {},
+  subscribe: () => {},
+};
+
+// mock Redux Slice functions
+jest.mock('../redux/rockets/rocketsSlice', () => ({
+  getRocketsRequest: jest.fn(),
+}));
+jest.mock('../redux/dragons/dragonsSlice', () => ({
+  fetchDragons: jest.fn(),
+}));
```

Comment:
I think this would be very useful for the rest of the team.
Great job @jlvFlores !!!

Reply:
I'm glad to hear that.